wikipedia/start.sh

```diff
@@ -24,7 +24,7 @@ kafka-topics --zookeeper localhost:2181 --topic WIKIPEDIANOBOT --create --replic
 kafka-topics --zookeeper localhost:2181 --topic EN_WIKIPEDIA_GT_1 --create --replication-factor 1 --partitions 2
 kafka-topics --zookeeper localhost:2181 --topic EN_WIKIPEDIA_GT_1_COUNTS --create --replication-factor 1 --partitions 2
 
-if is_ce; then confluent local config wikipedia-irc -- -d ./connector_irc.config; else confluent local config wikipedia-irc -d ./connector_irc_oss.config; fi
+if is_ce; then confluent local config wikipedia-irc -- -d ./connector_irc.config; else confluent local config wikipedia-irc -- -d ./connector_irc_oss.config; fi
```

Comment:
@sdanduConf FYI this line was fixed by 7a91b493 (committed 3 days ago).

Reply:
rebased, this was the only conflict and i kept what was in 5.4.x already